specs/interop/predeploys.md

```diff
@@ -94,19 +94,21 @@ The `ExecutingMessage` event represents an executing message. It MUST be emitted
 to `executeMessage`.
 
 ```solidity
-event ExecutingMessage(bytes,bytes);
+event ExecutingMessage(bytes32,bytes);
```

Comment:
What is the rationale for this change? It saves a little bit of gas but adds an extra RPC query for consumers of this to find the original log data, as they must look it up from the `Identifier`

Reply:
this is a good question @mds1 .
the rationale is the following:
the `ExecutingMessage` is designed to be used by the Interop service, which does not really care about the plain data.
for other off-chain agents wanting to consume the `bytes _data` argument, they can, as you said, do 1 RPC call to the Identifier origin domain, or they could also trace the tx (which is more complex).
remember data could get quite big, since it can be dynamically generated on-chain.

you raise a good point which is "how can we make off-chain devs life easier when having to deal with multiple RPCs", especially when we might end up with bundled txs that reference multiple Identifiers from different domains.
@fainashalts will be able to provide more insight on this last q' :)